src/ui/zcl_abapgit_gui_view_repo.clas.abap

```diff
@@ -104,7 +106,7 @@ CLASS zcl_abapgit_gui_view_repo IMPLEMENTATION.
 
     CREATE OBJECT ro_toolbar.
 
-    IF mo_repo->is_offline( ) = abap_false.
+    IF mo_repo->is_offline( ) = abap_false or mo_repo->has_remote( ) = abap_true.
```

Comment:
This IF checks if the repo is online or it has_remote, I guess online always has_remote? So it can be changed to only "mo_repo->has_remote( ) = abap_true"?

Reply:
The approach was to minimize change of existing logic. But yes, most probably has_remote is reliable enough. I'll adjust.